enhancements/insights/insights_gateway.md

```diff
@@ -0,0 +1,185 @@
+---
+title: insights-gateway
+authors:
+  - "@iNecas"
+reviewers:
+  - "@jhjaggars"
+  - "@chambridge"
+  - "@martinkunc"
+  - "@mfojtik"
+approvers:
+  - "@mfojtik"
+  - "@smarterclayton"
+creation-date: 2020-08-17
+last-updated: 2020-08-17
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Insights Gateway
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Insights-operator provides a proxy/gateway to provide a secure yet simple
+way for additional components to send payloads from the OCP cluster to
+cloud.redhat.com. This reduces complexity on user as well as developer
+side and adds additional control over the data sent to cloud.redhat.com.
+
+## Motivation
+
+In the last couple of releases, we've seen multiple new services build in 
+cloud.redhat.com (c.r.c.) providing additional insights from various areas,
+each needing additional data to be sent from the OCP clusters, such as:
+
+- [cost management](https://github.com/project-koku/korekuta-operator)
+- [subscription watch](https://github.com/chambridge/subscription-watch-operator)
+- [marketplace](https://github.com/RedHatInsights/marketplace-processor)
```

Comment:
This is the marketplace operator, what you linked is the service on cloud.redhat.com that processes the marketplace content:
https://github.com/redhat-marketplace/redhat-marketplace-operator/blob/acf4f3aaef03f15b6a88b0e2dcf8c3b52c3d7294/pkg/reporter/uploader.go

Reply:
Updated, thanks.